backend/test/policies.test.js

```diff
@@ -0,0 +1,122 @@
+const chai = require('chai')
+const chaiHttp = require('chai-http')
+chai.use(chaiHttp)
+const expect = chai.expect
+
+const { ROOT_BACKEND_URL } = require('./const')
+const {
+  getTestWallet,
+  createTestShop,
+  getOrCreateTestNetwork,
+  generatePgpKey
+} = require('./utils')
+
+describe('Shop policies', () => {
+  let shop
+
+  before(async () => {
+    //Reference: './order.test.js'
+    const ntwk = await getOrCreateTestNetwork()
+    const passphrase = 'abcdefghijklmnopqrstuvwxyz'
+    const key = await generatePgpKey('tester', passphrase)
+    const wallet = getTestWallet(1)
+    shop = await createTestShop({
+      network: ntwk,
+      sellerPk: wallet.privateKey,
+      pgpPrivateKeyPass: passphrase,
+      pgpPublicKey: key.publicKeyArmored,
+      pgpPrivateKey: key.privateKeyArmored
+    })
+  })
+
+  // testPolicies1, testPolicies2, testPolicies3 mimic the data supplied by the FE.
+  const testPolicies1 = [['', '', false]]
+
+  const testPolicies2 = [
+    [
+      'Terms and Conditions',
+      'Eget egestas purus viverra accumsan in nisl nisi scelerisque.',
+      false
+    ],
+    [
+      'Privacy Policy',
+      'Lorem ipsum dolor sit amet, consectetur adipiscing elit.',
+      false
+    ],
+    ['Return Policy', 'Eget egestas purus viverra accumsan.', false]
+  ]
+
+  const testPolicies3 = []
+
+  it('Should post new shop policies', async () => {
+    //If an admin just clicks on the 'Update' button without adding any policy, the server should just return a '200' response
+
+    await chai
```

Comment:
Thanks for adding those tests!
Sorry for being picky, but lould we keep the codebase consistent and use the same patterns as in all the other tests in the backend/tests directory and not used chained calls like await foo().request().post()...?

Reply:
Yes.